### DISCUSSION TITLE
[solved] Can docker-mailserver forward emails via the command ./setup.sh ?

### DISCUSSION THREAD

**Discussion by fab-kaz** [docker-mailserver/docker-mailserver#2825] [Q&A] [answered]
Hello all,

I guess the answer is "no" since i checked ;)

I created an email account "name@mydomain.org" with ./setup.sh : OK
I'd like to forward all mails who arrive to "name@mydomain.org" to another email "name2@riseup.net" using setup.sh

What to do you think of this ?

regards,

f.

**Comment by casperklein** [comment, marked as answer]
You can try an alias for this:

`./setup.sh alias add name@mydomain.org name2@riseup.net`

**Reply by fab-kaz** [reply]
Hello,

How could have I been so stupid ? of course, you're perfectly right!

thank you for answer and have a good day ;)

regards,

f.
